src/Runner.Worker/ActionRunner.cs

```diff
@@ -293,10 +287,14 @@ private string GenerateDisplayName(ActionStep action, DictionaryContextData cont
                 return displayName;
             }
             // Try evaluating fully
-            var templateEvaluator = context.ToPipelineTemplateEvaluator();
             try
             {
-                didFullyEvaluate = templateEvaluator.TryEvaluateStepDisplayName(tokenToParse, contextData, out displayName);
+                if (tokenToParse.CheckHasRequiredContext(contextData, context.ExpressionFunctions))
```

Comment:
"Try" was misleading since it could through. Refactored to generically reusable CheckHasRequiredContext